modules/cloud/pom.xml

```diff
@@ -33,7 +33,7 @@
     <url>http://ignite.apache.org</url>
 
     <properties>
-        <jcloud.version>1.9.0</jcloud.version>
+        <jcloud.version>2.0.0</jcloud.version>
```

Comment:
What is the reason of declaring the property here, and not to use it in the dependencies above? And why they are different? I think this is confusing at least, if no explanatory comments are added describing why it is that.